tests/mtest/src/main/scala/tests/TestHovers.scala

```diff
@@ -28,21 +27,7 @@ trait TestHovers {
   ): String = {
     hover match {
       case Some(value) =>
-        val types = value.getContents.asScala match {
-          case Right(value) =>
-            value.getValue
-          case Left(values) =>
-            values.asScala
-              .map { e =>
-                e.asScala match {
-                  case Left(value) =>
-                    value
-                  case Right(marked) =>
-                    codeFence(marked.getValue, marked.getLanguage)
-                }
-              }
-              .mkString("\n")
-        }
+        val types = value.getContents.getRight.getValue()
```

Comment:
the unhandled "left" case is the deprecated `MarkedString | String` case, which we never produce